cmd/lifecycle/analyzer.go

```diff
@@ -120,6 +120,30 @@ func (a *analyzeCmd) Privileges() error {
 		if err != nil {
 			return cmd.FailErr(err, "initialize docker client")
 		}
+	} else {
+		if a.platformAPIVersionGreaterThan06() {
```

Comment:
We'll need to do this for the `creator` also (see https://github.com/buildpacks/lifecycle/blob/771ac0b9d0d56cd3baaf28355498a0a636ea6182/cmd/lifecycle/creator.go#L261)

Reply:
I added the changes in the creator in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/fbdb97d0370bcbe1e9bdc6499c40839ee1255fc3)